cli/command/image/list.go

```diff
@@ -46,6 +47,7 @@ func NewImagesCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags.BoolVar(&opts.showDigests, "digests", false, "Show digests")
 	flags.StringVar(&opts.format, "format", "", "Pretty-print images using a Go template")
 	flags.VarP(&opts.filter, "filter", "f", "Filter output based on conditions provided")
+	flags.StringVar(&opts.sortby, "sort-by", "", "Customize list order, sort by field[:order]")
```

Comment:
Added some unit tests for sort package and one integration test instead. :smile: